actionview/lib/action_view/helpers/asset_tag_helper.rb

```diff
@@ -110,6 +113,9 @@ module AssetTagHelper
       #
       #   javascript_include_tag "http://www.example.com/xmlhr.js", defer: true
       #   # => <script src="http://www.example.com/xmlhr.js" defer="defer"></script>
+      #
+      #   javascript_include_tag "http://www.example.com/xmlhr.js", type: "module"
+      #   # => <script src="http://www.example.com/xmlhr.js" rel="modulepreload"></script>
```

Comment:
Is this correct? When I tried in a Rails console I got this:

```irb
$ rails c
Loading development environment (Rails 8.1.0.alpha)
min(dev)> helper.javascript_include_tag "http://www.example.com/xmlhr.js", type: "module"
=> "<script src=\"http://www.example.com/xmlhr.js\" type=\"module\"></script>"
```

From reading the code, it looks like `rel="modulepreload"` will get added to the `preload_links_header`, but I don't see how it would be added to the `<script>` in HTML.